en/flight_modes/offboard.md

```diff
@@ -61,21 +61,26 @@ bool actuator
-| velocity (NED)           |     &cross;    |     &check;    |          -         |        -       |        -        |        -        |      velocity     | `TrajectorySetpoint`                                                                                                              |
-| acceleration (NED)       |     &cross;    |     &cross;    |       &check;      |        -       |        -        |        -        |      velocity     | `TrajectorySetpoint`                                                                                                              |
+| position (NED)           |     &check;    |        -       |          -         |        -       |        -        |        -        |      position     | `TrajectorySetpoint.position`                                                                                                              |
+| velocity (NED)           |     &cross;    |     &check;    |          -         |        -       |        -        |        -        |      velocity     | `TrajectorySetpoint.velocity`                                                                                                              |
+| acceleration (NED)       |     &cross;    |     &cross;    |       &check;      |        -       |        -        |        -        |      velocity     | `TrajectorySetpoint.acceleration`                                                                                                              |
 | attitude (RFD)           |     &cross;    |     &cross;    |       &cross;      |     &check;    |        -        |        -        |        none       | [VehicleAttitudeSetpoint](../msg_docs/VehicleAttitudeSetpoint.md)                                                               |
 | body_rate (RFD)          |     &cross;    |     &cross;    |       &cross;      |     &cross;    |     &check;     |        -        |        none       | [VehicleRatesSetpoint](../msg_docs/VehicleRatesSetpoint.md)                                                                     |
 | thrust and torque (RFD)  |     &cross;    |     &cross;    |       &cross;      |     &cross;    |     &cross;     |     &check;     |        none       | [VehicleThrustSetpoint](../msg_docs/VehicleThrustSetpoint.md) and [VehicleTorqueSetpoint](../msg_docs/VehicleTorqueSetpoint.md) |
 
 where &check; means that the bit is set, &cross; means that the bit is not set and `-` means that the bit is value is irrelevant.
 
+:::note
+Generally you should only use the setting message and fields corresponding to the bit set in `OffboardControlMode`.
+If you set a field that injects an offboard setpoint at a lower level than the highest active controller, it may conflict with setpoints generated by a higher controller.

```

Comment:
So this is my attempt to explain why you should always set the bit to match the thing that you are controlling.

Even if you consider a position controller feeding into a velocity controller, there are potential conflicts right if you were to set the velocity but not position - since that position controller is active if the offboardcontrol bit is alive, and will be sending _some_ setpoint.